build.gradle

```diff
@@ -31,6 +31,8 @@ dependencies {
     // This dependency is found on compile classpath of this component and consumers.
     implementation 'com.google.guava:guava:27.0.1-jre'
 
+    compile files('src/main/java/dependencies/commons-codec-1.7.jar')
```

Comment:
Can you point this to a remote repo spec (https://mvnrepository.com/artifact/commons-codec/commons-codec/1.12#gradle)? Generally shouldn't check in jars into the repo.

Reply:
Done. Thanks for the suggestion.